doc/src/quickstart.md

```diff
@@ -0,0 +1,91 @@
+# Quickstart Guide
+
+This is a short guide to get you started. For a more detailed explanation about
+what's going on, please refer to the [tutorial](tutorial/index.html).
```

Comment:
I think we need to quickly mention the main terminology: Yacc and lex files, at least.

Reply:
Maybe I could do this in an Introduction section?

Reply:
I think a sentence or two in the quick start would be a good idea too. An ideal quick start section is enough to get semi-experienced users up and running on its own, so it needs to be fairly self contained.